tests/unit/cdr_data_dictionary/test_generate_yaml.py

```diff
@@ -100,30 +100,10 @@ def test_sequentially_process_tab_contents(self):
         self.assertEqual(fields, expected_fields)
         self.assertEqual(concepts, expected_vals)
 
-    def test_process_tab_contents(self):
-        # pre-conditions
-        values = [
-            ['field One', 'field Two', 'Field Three'],
-            ['val_one', 'val_two', 'val_three'],
-            ['val_four', 'val_five', 'val_six'],
-        ]
-
-        #test
-        fields, concepts = gen.process_tab_contents(values, 1)
-
-        # post conditions
-        expected_fields = ['field_one', 'field_two', 'field_three']
-        expected_vals = {
-            'val_two': {'field_one': ['val_one'], 'field_three': ['val_three']},
-            'val_five': {'field_one': ['val_four'], 'field_three': ['val_six']}
-        }
-
-        self.assertEqual(fields, expected_fields)
-        self.assertEqual(concepts, expected_vals)
-
     @patch('cdr_data_dictionary.generate_yaml.LOGGER')
     def test_write_yaml_file_list(self, mock_logging):
         # pre-conditions
+        self.maxDiff = None
```

Comment:
Can this be removed?

Reply:
could be removed.  but when the unit test failed, i had to add it to see the exact failure location in the diff.  it is not strictly necessary.  your recommendation?

Reply:
I've used tis too but have always deleted it after. It's not hurting anybody so we can leave it in